examples/simple/test_collection.json

```diff
@@ -0,0 +1,43 @@
+{
+	"info": {
+		"_postman_id": "f49e7563-d035-4164-baa9-0e77af3c6221",
+		"name": "testCollection",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Get",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "24c58423-6eb9-4ce5-9d28-b579702c91ef",
+						"exec": [
+							"pm.test(\"Status test\", function () {\r",
+							"    pm.response.to.have.status(200);\r",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "https://postman-echo.com/get",
+					"protocol": "https",
+					"host": [
+						"postman-echo",
+						"com"
+					],
```

Comment:
This might be a bit of a nit, but you might want to move the hostname over to `test_environment.json` since that's the best practice we are encouraging.

Reply:
Not sure I understand

Reply:
https://github.com/byu-oit/hw-lambda-api/blob/master/.postman/hw-lambda-api.postman_collection.json#L34-L42
https://github.com/byu-oit/hw-lambda-api/blob/master/.postman/dev-tst.postman_environment.json#L5-L9

Like I said, it's a bit of a nit. But it shows the reason you split some stuff out of the collection file into environment files.

Reply:
Ok, I get it now. Thanks for pointing it out. I updated the files in both examples